addons/libs/images.lua

```diff
@@ -407,6 +426,9 @@ windower.register_event('mouse', function(type, x, y, delta, blocked)
             if meta[dragged.image].root_settings then
                 config.save(meta[dragged.image].root_settings)
             end
+            if x == clicked.x and y == clicked.y then
```

Comment:
We are only registering left_click events for draggable images if the image wasn't dragged between click and release.